### ISSUE TITLE
CRAN 0.3.0 release

### ISSUE THREAD

**Issue by grantmcdermott** [grantmcdermott/tinyplot#280]
@vincentarelbundock @zeileis 

Just a quick update from my side. I've finally managed to wrap up some big end of year projects at work and am now hoping to submit a new version of tinyplot to CRAN before their Christmas break (23rd iirc).

Ideally, we can close (some of) the following issues:

- #243
- #271
- #278 
- #279

Regardless of where we get to, I plan to submit over the weekend. Hopefully, we'll get a straight pass through pre-checks etc. and v0.3.0 will be available as a little Christmas present to our users (and ourselves!)

Any thoughts / objections? If either of you have bandwidth to take on one of the above issue, please let me. Otherwise, I'll try to tick them off myself one by one.

**Comment by vincentarelbundock** [comment]
This sounds awesome!

Unfortunately, I won't have time to work on this before the break, as I'm finishing up a conference paper that needed to be sent out a while ago already...

**Comment by zeileis** [comment]
If you can pull this off: Great!! If not: Perfectly fine.

Personally, I try not to submit too much stuff (be it papers, reviews, packages, ...) before Christmas because the time is so busy for everybody anyway. Instead, I do it as one of the first things in the new year. Then I can start the year productively and my submission is likely to get more attention as well.

**Comment by grantmcdermott** [comment]
Looks like time slipped away from me... and now I'm sticking to the Zeileis philosophy of it being better to submit things in the new year :-D

**Comment by grantmcdermott** [comment]
@vincentarelbundock @zeileis Would either of you have any objections if I submit v0.3.0 to CRAN this week (or even tomorrow, if possible)?

While there's always more to do, I think that we've addressed most of the big ticket items for this release. I'm also heading out for a week+ of vacation Friday and would really like to have this taken care of before I unplug. (I had hoped to fit in a few extras like #303, but realistically I won't have time and it's fine to take them later.)

LMK. If everyone is happy then I'll fire off the submission :-)

**Comment by zeileis** [comment]
Fine for me!

**Comment by vincentarelbundock** [comment]
Absolutely!

**Comment by grantmcdermott** [comment]
**tinyplot** v0.3.0 now on CRAN 🎉🎉
https://cran.r-project.org/web/packages/tinyplot/index.html

Thanks both for your amazing work on this. I'm really stoked with how the package has shaped up and this genuinely feels like a milestone release.

**Comment by zeileis** [comment]
Yes, definitely! The types and themes have really taken the package to a different level. I'll try to do some more advertising as well because I think that people can really start to use this in a lot of applications now. 💪
